rrmngmnt/filesystem.py

```diff
@@ -89,3 +89,48 @@ def create_script(self, content, path):
                 raise errors.CommandExecutionFailure(
                     executor, cmd, rc, err,
                 )
+
+    def mkdir(self, path):
+        """
+        Create directory on host
+
+        :param path: directory path
+        :type path: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        return self.host.run_command(
+            ['mkdir', path]
+        )[0] == 0
+
+    def chown(self, path, username, groupname):
+        """
+        Change owner of file or directory
+
+        :param path: file or directory path
+        :type path: str
+        :param username: change user owner to username
+        :type username: str
+        :param groupname: change group owner to groupname
+        :type groupname: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        return self.host.run_command(
+            ['chown', '%s:%s' % (username, groupname), path]
+        )[0] == 0
+
+    def chmod(self, path, mode):
+        """
+        Change permission of directory or file
+
+        :param path: file or directory path
+        :type path: str
+        :param mode: permission mode(600 for example or u+x)
+        :type mode: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
```

Comment:
thanks that you raise exception instead of bools. please update doc as well.
and please add one negative test, and verify that exception contains expected message.